tutorials/image/cifar10/cifar10_multi_gpu_train.py

```diff
@@ -160,6 +161,12 @@ def train():
     # Create an optimizer that performs gradient descent.
     opt = tf.train.GradientDescentOptimizer(lr)
 
+    # Get images and labels for CIFAR-10.
+    # Force input pipeline to CPU:0 to avoid opertaios sometimes ending up on GPU
+    # and resulting in a slow down.
+    with tf.device('/CPU:0'):
```

Comment:
Since this is already in a block with tf.device('/cpu:0) you don't need to specify it again right?